docs/OG_Quickstart/OG_Quickstart.rst

```diff
@@ -72,6 +73,54 @@ Use the :meth:`~.glass.GlassPandas.rindex` method of the glass object to get the
     (1.5143223472613747, 1.5168000345005885, 1.5223762897312285)
 
 
+Defining a user-defined Glass object
+------------------------------------
+
+It is also possible to work with a user-defined Glass object. There are several types of Glass objects available. 
+One particularly useful class is :class:`~.opticalmedium.InterpolatedMedium`, which creates a glass object by interpolating between a finite number of (wavelength, refractive index) data points.
+
+For example, 'BD-2' is an infrared glass material whose refractive index is [tabulated](https://refractiveindex.info/download/data/2013/BD-2.pdf) by the manufacturer.
+To create a Glass object,
+
+
+.. code:: ipython3
+
+  from opticalglass import opticalmedium as om
+
+  # A list of ([wavelength in nm], [refractive index]) pairs
+  pairs = [
+    (3000, 2.6266), (4000, 2.6210), (5000, 2.6173), (6000, 2.6142), (7000, 2.6117),
+    (8000, 2.6088), (9000, 2.6055), (10000, 2.6023), (11000, 2.5983), (12000, 2.5942),
+    (13000, 2.5892), (14000, 2.5843)
+  ]
+  # user-defined glass object
+  glass = om.InterpolatedMedium('BD2', pairs=pairs, cat='LIGHTPATH')
+
+  # Then, this can be used as the same way to the predefined glass object
+  glass.rindex(2500)
+
+.. parsed-literal::
+  array(2.6306544)
+
+
+We can also *register* this user-defined glass by :func:`~.glassfactory.register_glass`
```

Comment:
I am not sure if this makes a valid link. Maybe there is some place where we specify which function is listed in readthedocs?